package.json

```diff
@@ -16,9 +16,7 @@
     "start:dev:watch": "yarn build && yarn serve",
     "gen-sql": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:generate -n",
     "migrate": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:run",
-    "gen-module": "IASQL_ENV=local ./.module_migration.sh",
-    "docker-compose:dev": "IASQL_ENV=local docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build",
-    "docker-compose": "IASQL_ENV=local docker-compose up --build"
```

Comment:
Not sure about removing this one. Are we sure this is not the one in use in the `CONTRIBUTING.md`?

Reply:
This is what we use in contributing. Removing this mostly for us, but will add back